gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -159,12 +159,23 @@ services:
     # frequency on which the notifier mechanism will test
     # if new notifications need to be send
     # default: 0 0 5 * * * (every day at 5am)
-    cronExpression: 0 0 5 * * *
+    cronExpression: 0 * * * * *
     # send notification if certificate is going to expire in less than 20 days, then 15 days before...
-    expiryThresholds: 20,15,10,5,1
+    expiryThresholds: 400,15,10,5,1
     # Subject of the email send by the email notifier
     expiryEmailSubject: Certificate will expire soon
 
+  # Rules about client's secrets expiry notifications.
+  # Require the platform notifier service.
+  client-secret:
+    enabled: true
+    # frequency on which the notifier mechanism will test
+    # if new notifications need to be send
+    # default: 0 0 5 * * * (every day at 5am)
+    cronExpression: 30 * * * * *
+    # send notification if client secret is going to expire in less than 20 days, then 15 days before...
+    expiryThresholds: 20,15,10,5,1
+
```

Comment:
I discussed internally about this settings with team members present when we introduced the cert notification. Your suggestion to move this settings (and the cert one) under the notification service make sens. So you can manage it, you just have to make sure about the backward compatibility to avoid issues.